uktrade/data-hub-frontend
# RR-727: React navigation on nunjucks

## Description of change

This makes use of DataHubHeader React component on pages still built using Nunjucks.

## Test instructions

Browsing Data Hub you should see absolutely no difference.

You may run Developer Tools and look up the class names on the Data Hub header - they should all appear random indicating the component is rendered from React.

Good page to look at is `/investments`.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
